Plots/Contours/NCL_conwomap_2.py

```diff
@@ -0,0 +1,66 @@
+"""
+NCL_conwomap_2.py
+===============
+This script illustrates the following concepts:
+   - Drawing a simple filled contour plot
+   - Selecting a different color map
+   - Changing the size/shape of a contour plot using viewport resources
+See https://www.ncl.ucar.edu/Applications/Scripts/conwomap_2.ncl for further information.
+"""
+
+###############################################################################
+# Import packages
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+from cartopy.mpl.geoaxes import GeoAxes
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+from util.make_byr_cmap import make_byr_cmap
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as tic
+import matplotlib.cm as cm
+
+###############################################################################
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset("../../data/netcdf_files/cone.nc")
+u = ds.u[4,:,:]
+
+###############################################################################
+# Contour map plot
+plt.rcParams['figure.figsize'] = [20, 15]
```

Comment:
minor: calling `fig.set_size_inches((20, 15))` is clearer?

Reply:
Agree and pushed the addressing changes.

Reply:
Agree and pushed the addressing changes.